### ISSUE TITLE
3.12 syntax warnings

### ISSUE THREAD

**Issue by kasium** [maurerle/eralchemy2#23]
With 3.12 some SyntaxWarning occurs:
```
venv/lib/python3.12/site-packages/eralchemy2/models.py:45: SyntaxWarning: invalid escape sequence '\*'
  '(?P<primary>\*?)(?P<name>[^\s]+)\s*(\{label:\s*"(?P<label>[^"]+)"\})?'
venv/lib/python3.12/site-packages/eralchemy2/models.py:115: SyntaxWarning: invalid escape sequence '\s'
  "(?P<left_name>[^\s]+)\s*(?P<left_cardinality>[*?+1])--(?P<right_cardinality>[*?+1])\s*(?P<right_name>[^\s]+)"
venv/lib/python3.12/site-packages/eralchemy2/models.py:200: SyntaxWarning: invalid escape sequence '\['
  RE = re.compile("\[(?P<name>[^]]+)\]")
```

**Comment by maurerle** [comment]
I had a fix pending in my workspace.
Which I did not yet test thoroughly.

Maybe you can give feedback if everything works fine on main?

**Comment by kasium** [comment]
Seems to run fine on my side

**Comment by maurerle** [comment]
As I released 1.3.8 - I close this issue then :)
